init.c

```diff
@@ -785,6 +830,8 @@ LUA_EXTERNC DLL_EXPORT int luaopen_libcutorch(lua_State *L);
 int luaopen_libcutorch(lua_State *L)
 {
   lua_newtable(L);
+  lua_pushvalue(L, -1);
+  lua_setglobal(L, "cutorch");
```

Comment:
Oh, just saw you changed that in init.lua. Is there a particular reason for that?